website/docs/api-layers.md

```diff
@@ -1267,21 +1267,25 @@ https://github.com/explosion/thinc/blob/master/thinc/layers/padded2list.py
 
 <inline-list>
 
-- **Input:** <tt>Sequence[Any]</tt>
+- **Input:** <tt>Union[Sequence[Hashable], Ints1d, Ints2d]</tt>
 - **Output:** <ndarray>Ints2d</ndarray>
 
 </inline-list>
 
-Remap string or integer inputs using a mapping table, usually as a preprocess
-before embeddings. The mapping table can be passed in on input, or updated after
-the layer has been created. The mapping table is stored in the `"mapping_table"`
-attribute.
+Remap a sequence of strings, integers or other Hashable inputs using a
+mapping table, usually as a preprocessing step before embeddings. The
+input can also be a two dimensional integer array in which case the
+`column: int` attribute tells the `remap_ids` layer which column of the
+array to map. For 2D inputs the `column` has to be set on initialization.
+The mapping table can be set at initialization, but can also be passed in on input, 
```

Comment:
I don't think it can be passed in on input (or I don't know what you mean by "passed in on input")?

Something more like "can also be set or modified at a later point by setting the `mapping_table` attribute."